tools/loader/ImageClassifier.cpp

```diff
@@ -101,6 +112,24 @@ static bool getNextImageFilenames(std::vector<std::string> *filenames) {
   return !filenames->empty();
 }
 
+/// Generate in \p imageList the list of filenames corresponding to the next
+/// mini-batch of size \p miniBatchSize extracted from \p totalImageList at
+/// index \p minibatchIndex.
```

Comment:
Can you also update the doxygen here to include `\returns`, and also mention that `minibatchIndex` is updated?

Reply:
Done